pkg/stdlib/pkgs/query.cue

```diff
@@ -50,6 +50,9 @@
 		deployVersion?:   string
 		revision?:        string
 		latest?:          bool
+		resourceTree?: {
```

Comment:
Why add this field here?  `listAppliedResources` means only query the resources that render from the application.

Reply:
The struct was added by this value internally